### DISCUSSION TITLE
[Community Extensions] Overview

### DISCUSSION THREAD

**Discussion by kratzert** [kratzert/Caravan#10] [Announcements]
The purpose of this thread is to maintain a list of all community extensions to Caravan. We will update this list, once an author opens an issue to present a new extension.

- Denmark (308 basins) by [Julian Koch](https://github.com/JulKoch): [Issue with further details](https://github.com/kratzert/Caravan/issues/9), [Download link](https://doi.org/10.5281/zenodo.6645861)
- Israel (95 basins) by [Efrat Morin](https://github.com/EfratMorin): [Issue with further details](https://github.com/kratzert/Caravan/issues/18), [Download link](https://doi.org/10.5281/zenodo.7758515)
- Switzerland/CAMELS-CH (and adjacent basins) (296 basins) by [Marvin Höge](https://github.com/marv-in) et al.: [Issue with further Details](https://github.com/kratzert/Caravan/issues/21), [Download link](https://zenodo.org/records/15025258)
- Spain/CAMELS-ES (269 basins) by [Casado Rodríguez, Jesús](https://github.com/casadoj): [Issue with further Details](https://github.com/kratzert/Caravan/issues/25), [Download link](https://doi.org/10.5281/zenodo.8373020)
- Iceland/LamaH-Ice (88 basins) by [Hordur Helgason](https://github.com/hhelgason): [Issue with further Details](https://github.com/kratzert/Caravan/issues/27), [Download link](http://www.hydroshare.org/resource/705d69c0f77c48538d83cf383f8c63d6/)
- GRDC (25 countries, 5357 basins) by Claudia Färber, Henning Plessow, and Ulrich Looser (all GRDC) and Frederik Kratzert, Nans Addor, and Guy Shalev. [Issue with further Details](https://github.com/kratzert/Caravan/issues/26), [Download link](https://doi.org/10.5281/zenodo.8425586)
- Germany/CAMELS-DE (1887 basins) by [Alexander Dolich](https://github.com/AlexDo1), Ashim Maharjan, Mirko Mälicke, Ashish Manoj, Ralf Loritz [Issue with further Details](https://github.com/kratzert/Caravan/issues/35), [Download link](https://doi.org/10.5281/zenodo.13320514)
- MultiMet (additional nowcast + forecast data for all basins) by Guy Shalev and Frederik Kratzert. [Technical Documentation](https://arxiv.org/abs/2411.09459), [Nowcast data](https://zenodo.org/records/14161235), [Forecast data](https://zenodo.org/records/14161281). All data is also hosted on a public GCS bucket. For details, check [this example notebook](https://github.com/kratzert/Caravan/blob/main/examples/Caravan_MultiMet_Extending_Caravan_with_Multiple_Weather_Nowcasts_and_Forecasts.ipynb)
- Czechia/CAMELS-CZ (249 basins) by CHMI,  Krejčí, Jakub, and Nearing, Grey [Issue with further Details](https://github.com/kratzert/Caravan/issues/49), [Download link](https://doi.org/10.5281/zenodo.17338510)
- Australia (Maribyrnong River)/ (10 gauges) by [maribyrnongfloods](https://github.com/maribyrnongfloods) and [Lee Lanzafame](https://github.com/llanzafame)[Issue with further Details](https://github.com/kratzert/Caravan/issues/51), [Download link](https://doi.org/10.5281/zenodo.18736843)
- CMIP6 (bias-corrected climate model projections for all basins) by [Sacha Ruzzante](https://github.com/sruzzante). [Technical Documentation](https://doi.org/10.5194/essd-2026-350), [Download Link](https://doi.org/10.20383/103.01644). All data is also hosted on a public GCS bucket. For details, check [this example notebook](https://github.com/sruzzante/Caravan-CMIP6/blob/main/Caravan_CMIP6_Colab_example.ipynb)

**Comment by grey-nearing** [comment]


**Reply by kratzert** [reply]
Thanks for the info. Note that the Download links above are those special Zenodo links that always point to the newest version. So no change is required from our end.

**Comment by dajimenezo** [comment]
Dear Prof. Kratzer,

We have prepared CARAVAN-COL, which is now available in the following repository: https://doi.org/10.5281/zenodo.21629906
